### PR TITLE
Ads panel

### PR DESCRIPTION
**Pull request by Maymonakh**

### CODE REVIEW

File: app/components/AdsPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Box, Text } from "@chakra-ui/react";
```
Comment (talaween-net) [inline app/components/AdsPanel.tsx:2]: no need for chakra UI but it is okay

File: app/components/AdsPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+const AdPanel: React.FC<AdPanelProps> = ({ startDate, endDate, adText }) => {
+  const now = new Date();
+  const start = new Date(startDate);
+  const end = new Date(endDate);
+  const isAdActive = now >= start && now <= end;
+
+  return (
+    <>
```
Comment (talaween-net) [inline app/components/AdsPanel.tsx]: should return null not <></> (-1)

File: app/page.tsx [hunk 3]
```diff
@@ -1,95 +1,14 @@
-import Image from "next/image";
-import styles from "./page.module.css";
+import AdPanel from "./components/AdsPanel";
+import { ChakraProvider } from "@chakra-ui/react";
 
 export default function Home() {
   return (
-    <div className={styles.page}>
```
Comment (talaween-net) [inline app/page.tsx:6]: too many unrelated code in the same PR (-2)

File: app/components/AdsPanel.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      {isAdActive && (
+        <Box className={styles.adPanel}>
+          <Text >{adText}</Text>
+        </Box>
+      )}
+    </>
+  );
+};
+
+export default AdPanel;
```
Comment (talaween-net) [inline app/components/AdsPanel.tsx]: companent name is not the same as the file name (-1)

**Review by talaween-net — COMMENTED** [review]
final score 6/10

**Review by NadaObaid1 — APPROVED** [review]
